app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const { getCategories } = require("./controllers/categories-controllers.js");
+const { handle500Statuses } = require("./controllers/error-handling-controllers.js");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
You shouldn't have this line until you're actually using the function of expess.json